### DISCUSSION TITLE
Chatops action-alias for execute Workflow

### DISCUSSION THREAD

**Discussion by antonbezkrovny** [StackStorm/st2#5624] [ChatOps] [answered]
Hello everyone!
 I can't find in the documentation how to use action-alias to start workflow
For alias, I can specify action_ref and formats in the yaml config

How can I do the same for workflow?

**Comment by nzlosh** [comment, marked as answer]
workflows are actions, so you would just write a workflow and then reference it as you would any other action.  As an example, if you used a custom pack to keep your workflows called `work_tasks` and you wrote a workflow called `remove_old_logs`.  It would be defined in the action-alias as:
```
action_ref: work_tasks.remove_old_logs
```
